style/icons/shield40_us_dc.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="22" xmlns="http://www.w3.org/2000/svg">
+ <rect width="19" height="21" rx="1" ry="1" x=".5" y=".5" style="fill:#fff;fill-opacity:1;fill-rule:nonzero;stroke:#000;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none"/>
+ <path style="fill:#000;stroke-width:.0311206" d="M 7.8554688 0.81445312 L 0.91601562 7.3222656 L 1.2285156 7.640625 C 1.5054406 7.9444685 2.298937 8.5951539 2.4023438 8.6679688 C 2.4219668 8.6817868 2.549775 8.7535573 2.6484375 8.8125 C 2.866025 8.942711 3.1822781 9.057752 3.4707031 9.109375 C 3.9903906 9.202425 4.595 9.3512987 4.71875 9.4179688 C 4.79275 9.4578488 4.8841844 9.4921875 4.9199219 9.4921875 C 5.0197969 9.4921875 5.3380094 9.7895721 5.3652344 9.9082031 L 5.3867188 10 L 6.234375 10 L 6.2109375 9.9121094 C 6.1159375 9.5684974 5.687025 9.0737033 5.4375 9.0195312 C 5.37025 9.0049313 5.183875 8.9442797 5.0234375 8.8867188 C 4.8629875 8.8291897 4.6653594 8.7730696 4.5839844 8.7597656 C 4.2142719 8.6523356 3.5659406 8.5348755 3.4472656 8.4765625 C 3.0706406 8.4264435 2.2089844 7.6777344 2.2089844 7.6777344 L 1.9882812 7.4472656 L 7.8417969 1.9570312 L 8.1582031 1.6621094 L 8.3789062 1.8398438 L 17.955078 9.5058594 L 17.220703 10.320312 L 17.878906 10.775391 L 19.083984 9.4433594 L 8.3769531 0.81445312 L 7.8554688 0.81445312 z M 6.8632812 18.5 C 6.9010696 18.617853 6.9817812 18.773886 7.1582031 19.105469 C 7.2162031 19.214468 7.3256562 19.499061 7.3554688 19.603516 C 7.4105938 19.818169 7.4181719 19.920993 7.4511719 20.140625 C 7.4917969 20.366827 7.5409219 20.63546 7.5605469 20.738281 C 7.5801719 20.841135 7.5979844 20.980042 7.5996094 21.046875 L 7.6015625 21.185547 L 8.421875 21.177734 L 10.859375 18.5 L 9.8222656 18.5 L 8.4199219 20.048828 L 8.2792969 20.205078 L 8.2167969 19.990234 C 8.1871719 19.889123 8.1490625 19.687784 8.1328125 19.544922 C 8.1140625 19.379452 8.0613281 19.210396 7.9863281 19.078125 C 7.9215781 18.963929 7.8488812 18.810257 7.8242188 18.738281 C 7.8045745 18.680828 7.7643641 18.583809 7.7304688 18.5 L 6.8632812 18.5 z"/>
```

Comment:
This is a tough one. For Rebusurance, I wound up whittling the district’s shape down to nothing more than a pair of right angles pointing up and down. Increasing the shield height is a good way to avoid that fate, though it might be possible to trim a little more of the Potomac riverbank without sacrificing recognizability.